Dockerfile

```diff
@@ -1,15 +1,28 @@
-FROM ipython/scipystack
+FROM ubuntu:14.04
```

Comment:
:point_up: You're planning on replacing this with a Debian image, yes? They're much slimmer with little lost functionality.